### ISSUE TITLE
[Bug] dynamic island always active

### ISSUE THREAD

**Issue by LenardValdez** [qnblackcat/uYouPlus#658]
Device info

iOS/iPadOS version: iOS 16.1
Device model: iPhone 14 pro
Sideload tool: Signulous
The specific version of uYouPlus: 2.1
Describe the bug

When you play a video, the dynamic island is also active as if its playing an audio in the background  whilst the video is actually playing
![image](https://user-images.githubusercontent.com/23581438/200327671-dd99ae41-8976-4893-ac6b-74f9ed88e714.png)

How to reproduce the bug

Play any video and you can see that the dynamic island is also active

Are you using the newest version of uYouPlus? If not, why?

Yup, i just updated

Does the issue happen with the official YouTube from AppStore?

Nope

**Comment by MarioMastr** [comment]
Pretty sure that's intentional on iOS's end.

**Comment by LenardValdez** [comment]
> Pretty sure that's intentional on iOS's end. 

No i don't think so. On the non Modded YT it only appears when I background play a video

**Comment by MarioMastr** [comment]
> > Pretty sure that's intentional on iOS's end. 
> 
> No i don't think so. On the non Modded YT it only appears when I background play a video

Yeah that's my point. Background play involves the Now Playing API on iOS, and on 14 Pros, the API involves a viewer on the Dynamic Island similarly to the picture you have whether you like it or not. Regular YouTube iOS app doesn't have background play without Premium, so of course, this issue doesn't exist on the app. It isn't limited to uYouPlus. Try playing a YouTube video in safari and you will see something similar.

**Comment by LenardValdez** [comment]
> > > Pretty sure that's intentional on iOS's end. 
> > 
> > No i don't think so. On the non Modded YT it only appears when I background play a video
> 
> Yeah that's my point. Background play involves the Now Playing API on iOS, and on 14 Pros, the API involves a viewer on the Dynamic Island similarly to the picture you have whether you like it or not. Regular YouTube iOS app doesn't have background play without Premium, so of course, this issue doesn't exist on the app. It isn't limited to uYouPlus. Try playing a YouTube video in safari and you will see something similar. 

I see. But i do have premium. I just need the sponsorblock and the dislikes. I guess it overrides the default way of playing the video.

**Comment by qnblackcat** [comment]
Any update?

**Comment by LenardValdez** [comment]
> Any update?

Still getting that active dynamic island

**Comment by neoighodaro** [comment]
I second this as a bug. When using Cercube this bug doesn’t exist just noticed it only on uYouPlus. If that helps.

**Comment by neoighodaro** [comment]
It would appear it’s somehow related to Sideloadly. Used AltStore and this bug doesn’t exist.

**Comment by LenardValdez** [comment]
> It would appear it’s somehow related to Sideloadly. Used AltStore and this bug doesn’t exist.

Did you opt to remove unused plugins?

**Comment by neoighodaro** [comment]
No I did not. Always left it as defaults.

**Comment by GrPK** [comment]
Do you guys ever find a solution to this problem? Facing it right now (15PM). Having YT Premium

**Comment by neoighodaro** [comment]
Install via literally any other method than sideloadly

**Comment by GrPK** [comment]
I tried with both Scarlet and Signulous and have the same results

**Comment by neoighodaro** [comment]
Then use another build. I don't have this problem anymore but had it when i used sideloadly.

**Comment by LenardValdez** [comment]
> I tried with both Scarlet and Signulous and have the same results
> 
> P.S: I have a paid certificate (from Signulous) for notification

Still getting the active dynamic bar even on the latest build. Signulous might be doing something. Idk I came to accept it at this point hahahah

**Comment by hi-im-karma** [comment]
Having this exact same issue. Used MapleSign paid.

**Comment by hi-im-karma** [comment]
> Still getting that active dynamic island

Did you find a solution to this extremely bothering issue? I'm also a YouTube Premium member, and have the exact same issue with uyou+ and uyouenhanced...

**Comment by hi-im-karma** [comment]
> Do you guys ever find a solution to this problem? Facing it right now (15PM). Having YT Premium

Did you find a solution to this problem? I'm also YT Premium user and reported this issue numerous times: https://github.com/qnblackcat/uYouPlus/issues/1309

**Comment by neoighodaro** [comment]
Like I mentioned before, using Altstore + YouTubeRebornPlus_18.14.1_4.0.10.ipa (not from this repo) helped

**Comment by LenardValdez** [comment]
> > Do you guys ever find a solution to this problem? Facing it right now (15PM). Having YT Premium
> 
> Did you find a solution to this problem? I'm also YT Premium user and reported this issue numerous times: https://github.com/qnblackcat/uYouPlus/issues/1309

Apparently the fix requires the right entitlements on signing(think of these as the permissions of an app to access certain level of security i.e. playback visibility or lack thereof on the Dynamic Island, screen share, notifications, etc... this is a security level apple controls heavily in a way as these things could be used maliciously). self signing with your own iCloud account automatically grants you the correct entitlements, apple grants this to self signing as it assumes that you developed this app and you know what you are doing and does not restrict you with the permissions. Tldr; self sign is the fix for modded youtube ipas

**Comment by hi-im-karma** [comment]
It's just uyou developers not developing these modded YT apps accordingly for iPhones with Dynamic Island. They're not taking it into account.
